### PR TITLE
To do list : Add and Remove

### PR DESCRIPTION
**Pull request by sheylaPozo**
- [x] Use webpack to bundle JavaScript.
- [x] Remove all hardcoded items from the tasks array.
- [x] Create a new JavaScript file for the new functionality.
- [x] All changes to the To-Do List should be saved in local storage.
- [x] Fixed required changes of TSE specifications.

### CODE REVIEW

File: src/dragAndDrop.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+  const sourceObj = savedList[source];
+  const souceIndex = savedList[source].index;
+  let targetIndex;
+  // eslint-disable-next-line array-callback-return
+  savedList.map((obj) => {
+    if (obj.index === Number(target)) {
+      targetIndex = savedList.indexOf(obj);
+    }
+  });
```
Comment (henatan99) [inline src/dragAndDrop.js]: - [x] Please remove this eslint-disable command and give return a value for the savedList.map function.

File: src/addAndRemove.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        statusUpdate(e, i);
+        this.displayAllTask();
+      });
+
+      const p = document.createElement('p');
+      const index = document.createElement('spand');
+      index.className = 'index';
+      const button = document.createElement('i');
+      button.classList.add('fas', 'fa-ellipsis-v', 'icon');
+      p.addEventListener('keydown', (e) => {
```
Comment (henatan99) [inline src/addAndRemove.js:86]: - [x] I don't think the 'keydown' event is right here. Just key down doesn't make the description editable. There is a bug that doesn't let the description text be edited unless the `i` tag is clicked. Please, kindly fix this bug.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @sheylaPozo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] Please try to add a more descriptive summary to your PR. :pray: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/dragAndDrop.js (hunk 1 above)
Comment (sheylaPozo) [reply src/dragAndDrop.js]: ok fixed it

File: src/addAndRemove.js (hunk 2 above)
Comment (sheylaPozo) [reply src/addAndRemove.js:86]: Fixed it

**Review by de-scholar — APPROVED** [review]
Hi @sheylaPozo :handshake:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

:warning: Anyways your live demo link from the readme file is showing a different page (it displays your readme file) kindly fix it in order to make your repo more professional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
Thank you :) I will

**Pull request merged**